crates/wash-cli/src/app/mod.rs

```diff
@@ -95,14 +99,18 @@ pub struct DeployCommand {
 pub struct DeleteCommand {
     /// Name of the application to delete, or a path to a Wadm Application Manifest
     #[clap(name = "name")]
-    app_name: String,
+    app_name: Option<String>,
```

Comment:
same comment re: clap parsing

Reply:
Fixed this like the other one -- thanks again!